client/src/utils/dateTimeHelper.ts

```diff
@@ -149,6 +149,21 @@ export const formatDate = (
   return momentDate.isValid() ? momentDate.format(format) : ''
 }
 
+/**
+ * Format a date string into "MMM DD, YYYY • relative".
+ * Returns 'N/A' for null/invalid inputs to match UI expectations.
+ */
+export const formatDateWithRelative = (raw?: string | null): string => {
+  if (raw == null || (typeof raw === 'string' && raw.trim() === '')) return 'N/A'
```

Comment:
Simplify empty check. `!raw` already covers null, undefined, and empty strings after trimming.

```suggestion
 if (!raw || (typeof raw === 'string' && !raw.trim())) return 'N/A'
```